src/main/resources/documentation/domain-model-overview.md

```diff
@@ -0,0 +1,344 @@
+#What is the OpenHDS? 

```

Comment:
I like what you wrote here.  I realized I have never read a concise description of the project like this!